in_progress/roles.md

```diff
@@ -0,0 +1,109 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
```

Comment:
I'm confused by the language in this section. 

- Any new contributor to scvi-tools would need to be approved by two core devs from separate employers? 
- What rights does a contributor have?

Reply:
We did touch on this at a meeting right after, but I'll update with my POV here:

> What rights does a contributor have?

Not too much really. This role isn't about rights and responsibilities, but recognition for people who contribute – who don't quite rise to "core". We would add them to a team on the `scverse` governance repo. We could definiltey consider more, but it's a badge next to their name.

> Any new contributor to scvi-tools would need to be approved by two core devs from separate employers?

For contributing code and getting commit rights, no. But we could keep seperate out project developers and contributors.